generators/app/templates/client/server.js

```diff
@@ -0,0 +1,22 @@
+require('./config/env');
+var path = require('path');
+var url = require('url');
+var proxy = require('proxy-middleware');
+var history = require('connect-history-api-fallback');
+var serveStatic = require('express').static;
+var server = require('express')();
+
+var port = process.env.NODE_ENV == 'production' ? process.env.PORT : 8080;
+
+server.use(history());
+server.use('/', serveStatic(path.join(__dirname, 'public')));
+
+if(process.env.NODE_ENV !== 'production') {
+  server.use('/', proxy(url.parse(process.env.WEBPACK_DEV_SERVER_URL)))
+}
+
+server.use('/', proxy(url.parse(process.env.API_URL)));
```

Comment:
Two apps overkill, serve with rails is not an option too, express proxy is way lighter. I was worried because `heroku -> express -> rails` would add too much load to the server.

And in case of other apps try to use our api, this could cause some problem?